core/rawdb/accessors_skipped_txs.go

```diff
@@ -197,6 +197,11 @@ func WriteSkippedTransaction(db ethdb.Database, tx *types.Transaction, traces *t
 	}
 }
 
+func ResetSkippedTransactionTracesByHash(db ethdb.Database, txHash common.Hash) {
```

Comment:
Add comments to `ResetSkippedTransactionTracesByHash`.

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/501/commits/aced467ab596ee2594fabce510e3613c234a1370